modules/get-started/pages/whats-new.adoc

```diff
@@ -11,22 +11,33 @@ See also: xref:deploy:deployment-option/cloud/whats-new-cloud.adoc[]
 xref:develop:config-topics.adoc#configure-write-caching[Write caching] is a relaxed mode of `acks=all` that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write. For clusters in development mode, write caching is enabled by default. For clusters in production mode, it is disabled by default.
 
 == RBAC
-xref:manage:security/authorization/rbac.adoc[Role-based access control] (RBAC) provides an extension to Redpanda's existing access control lists (ACLs). It allows you to create roles and assign those roles as the authorized or denied party for an ACL. Every user you assign to that role inherits those authorizations. Using RBAC gives you a more flexible and more efficient tool to manage authorization for your users, especially when you have complex organizational hierarchies, large numbers of users, or both.  
+xref:manage:security/authorization/rbac.adoc[Role-based access control] (RBAC) provides an extension to Redpanda's existing access control lists (ACLs). Using `rpk` or Redpanda Console, you can create roles and assign those roles as the authorized or denied party for an ACL. Every user you assign to that role inherits those privileges. RBAC provides a more flexible and efficient way to manage user privileges, especially with complex organizational hierarchies or large numbers of users.  
 
 == Data transforms
 
-* Output to multiple topics: Data transforms can now output to multiple topics, for use cases that might require distributing different message types to multiple event streams, or for writing valid records to an output topic and invalid records to a dead-letter queue. See the xref:reference:data-transform-golang-sdk.adoc[Go] or xref:reference:data-transform-rust-sdk.adoc[Rust] SDK reference for data transforms for implementation details.
+* Output to multiple topics: Data transforms can now output to multiple topics.  For example, you can distribute different message types to multiple event streams, or you can write valid records to an output topic and invalid records to a dead-letter queue. See the xref:reference:data-transform-golang-sdk.adoc[Golang SDK] and xref:reference:data-transform-rust-sdk.adoc[Rust SDK] for data transforms.
```

Comment:
Seems odd that this is 2 bullets with no introduction. Why not use the actual intro, which does a nice job explaining what it is...
Redpanda provides a framework to build and deploy inline transformations, also known as data transforms, on data written to Redpanda topics, delivering processed and validated data to consumers in the format they expect. Redpanda does this directly inside the broker, eliminating the need to manage a separate stream processing environment or use 3rd-party tools. Data transforms enable you to run common data streaming tasks, like filtering, scrubbing, and transcoding, within Redpanda.

And then maybe put the existing two bullets below?

Reply:
added intro sentence